### ISSUE TITLE
Unable to choose from photo gallery (for attaching to message) while VoiceOver is enabled - iOS bug?

### ISSUE THREAD

**Issue by digitalken** [department-of-veterans-affairs/va.gov-team#23602]
## What happened?
With VoiceOver enabled, I'm unable to select a photo from the photo gallery to include as an attachment.  No matter which photo I tap on, VoiceOver goes back to the Cancel button and announces "Cancel button."

What makes me think this might be an iOS issue rather than our app is I can try to do a similar operation by trying to email myself a screen shot from the photo gallery and I have the same difficulty as mentioned above.

## Specs:
- Device: iPhone XR, VoiceOver enabled

## Steps to Reproduce
- Enable VoiceOver
- Compose a message and choose to include an attachment
- Choose the Photo Gallery
- When the Photo Gallery opens, try to select any photo.
-->  The photo will not be selected and the focus snaps back back to the Cancel button.

## Desired behavior
User is able to select a photo from the photo gallery with VoiceOver enabled.

## Acceptance Criteria
User is able to select a photo from the photo gallery with VoiceOver enabled.

## Bug Severity
- 2 - High

## Linked to Story
https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/22610
